### DISCUSSION TITLE
Do server files auto update when container starts?

### DISCUSSION THREAD

**Discussion by Panderine** [mbround18/valheim-docker#230] [Help!]
I need the server files to "stay put" and manually update when I choose so. Is this possible or the server files update every time the container is started?

**Comment by mbround18** [comment]
if you don't have the server installed, it will install the server. Otherwise the server will try to auto update each start. To turn this feature off add the following to your env: 

```sh
UPDATE_ON_STARTUP=0
```

Otherwise, it wont mess with the server files. Odin only sets various environment variable to launch it as a daemon.
